bin/test.py

```diff
@@ -1,16 +1,19 @@
 import os
-import subprocess
+from pathlib import Path
 
+import docker
 import pytest
 import testinfra
 import yaml
 
 ASTRO_IMAGE_NAME = os.environ["ASTRO_IMAGE_NAME"]
+ASTRO_IMAGE_TAG = os.environ.get("CIRCLE_SHA1", "latest")
```

Comment:
Just to be consistent with other places
```suggestion
ASTRO_IMAGE_TAG = os.getenv("CIRCLE_SHA1", "latest")
```